src/main/java/com/hazelcast/kubernetes/HazelcastKubernetesDiscoveryStrategy.java

```diff
@@ -53,6 +54,7 @@
 
     private static final String DEFAULT_MASTER_URL = "https://kubernetes.default.svc";
     private static final int DEFAULT_SERVICE_DNS_TIMEOUT_SECONDS = 5;
+    private static final int DEFAULT_KUBERNETES_API_RETRIES = 3;
```

Comment:
Do you think we should add some bound control... like negative values or too big values?

Reply:
Good idea, but let's do it together with other parameter validation, I've created a separate issue for that: #135